ui/scripts/package.sh

```diff
@@ -13,12 +13,13 @@ rm -rf $ROOT_DIR/target/${ZIP_FILE_NAME}*.zip
 npm install
 bower install
 
-pids=$(pgrep Xvfb)
-if [ -n "$pids" ]; then
+if [ -z $(pgrep Xvfb) ]; then
```

Comment:
Putting the pgrep directly within the test prevents a non-zero exit code from prematurely terminating the script.

Also note that I believe we should be testing with -z here, not -n, since we want to return true if the value DOES have zero size.